pkgs/kde/misc/klevernotes/default.nix

```diff
@@ -0,0 +1,25 @@
+{
+  lib,
+  mkKdeDerivation,
+  fetchurl,
+  kio,
+  qtsvg,
+  qtwebengine,
+}:
+mkKdeDerivation rec {
+  pname = "klevernotes";
+  version = "1.0.0";
+
+  src = fetchurl {
+    url = "mirror://kde/stable/klevernotes/${version}/klevernotes-${version}.tar.xz";
+    hash = "sha256-9nXOaa2/DyJO3nXGbnvPaYjXFrVoAgx52th67l5geOU=";
+  };
+
+  extraBuildInputs = [
+    kio
```

Comment:
kio should already be injected automatically from metadata.

Reply:
You’re right, I had an issue with CMake not finding it yesterday, but today’s build without it works fine. Removed it in the last push